proto/trip.proto

```diff
@@ -272,6 +272,11 @@ message TripLeg {
       uint32 end_shape_index = 2;
     }
   };
+  message Summary {
```

Comment:
I don't know what else to call it, though `Summary` doesn't make much sense without `time`, `length` etc from the equivalent one in directions.proto. or maybe we should put the one from directions.proto into `common.proto` and import it in both places. but then for trip legs we only use this sub-set of attributes to check whether a leg encounters toll etc?

Reply:
I think I'd prefer that. WDYT @kevinkreiser @dnesbitt61 ?

Reply:
using the same name with a different meaning is really not great. why can't the be top level in the leg? also isn't this info technically already available in the edges of the leg?

Reply:
oh isee directions summary does have this info

Reply:
Yeah and that gets it from the maneuvers, which is why we have that bug. We needed a way to bring the info from thor to odin so I suggested this way.

should we just pull out the „Summary“ message to common.proto and let both TripLeg and DirectionsLeg share it?

Reply:
yep sounds good lets share it from common.proto

Reply:
Ok, thanks for suggestions, done